SharedProject/SharedSource/powerConnections.cs

```diff
@@ -22,6 +22,7 @@ public partial class Plugin : IAssemblyPlugin
             "steam",
```

Comment:
This entire file needs to be merged into the Plugin.cs file under the Shared Project.